File: baremetal/arch/aarch64/add.py
def test(self):
    self.sendline('tbreak main')
    self.sendline('continue')
    self.continue_to('op1')
    assert self.get_int('$x0') == 1
    self.continue_to('result')
    assert self.get_int('$x1') == 3

File: baremetal/add.c
#include <common.h>

void main(void) {
	int i, j, k;
	i = 1;
    /* test-gdb-op1 */
	j = 2;
    /* test-gdb-op2 */
	k = i + j;
    /* test-gdb-result */
	if (k != 3)
		assert_fail();
}
